red-hat-data-services/odh-manifests
# Add external images directly to params for dsp.

Resolve: https://github.com/red-hat-data-services/odh-manifests/issues/424

# Testing instructions: 
Create a live build image from this fork and change in odh-deployer, and confirm dspo/dspa's are deployed with the appropriate images.

Live builder image: 
Image: quay.io/hukhan/rhods-operator-live-catalog:1.31.0-3

Dashboard image used: quay.io/hukhan/managed-open-data-hub-odh-dashboard-rhel8:v1.31.0-1 
<details>

<summary>upstream_sources.yaml</summary>

```yaml
git:
  # ODH Deployer
  - name: odh-deployer-container-live
    url: https://github.com/HumairAK/odh-deployer
    branch: rm_images
    patch: rhods-operator-live/dockerfiles/odh-deployer/Dockerfile

  # ODH Operator Base
  - name: odh-operator-base-container-live
    url: https://github.com/red-hat-data-services/opendatahub-operator
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-operator-base/Dockerfile

  # ODH Operator
  - name: odh-operator-container-live
    url: https://github.com/HumairAK/odh-manifests
    branch: add_imgs
    patch: rhods-operator-live/dockerfiles/odh-operator/Dockerfile

  # ODH Dashboard
  - name: odh-dashboard-container-live
    url: https://github.com/red-hat-data-services/odh-dashboard
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-dashboard/Dockerfile

  # Kubeflow Notebook Controller
  - name: odh-kf-notebook-controller-container-live
    url: https://github.com/red-hat-data-services/kubeflow
    branch: master
    context: components
    containerfile: notebook-controller/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-kf-notebook-controller/Dockerfile

  # ODH Notebook Controller
  - name: odh-notebook-controller-container-live
    url: https://github.com/red-hat-data-services/kubeflow
    branch: master
    context: components
    containerfile: odh-notebook-controller/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-notebook-controller/Dockerfile

  # ODH ML Pipelines Persistence Agent
  - name: odh-ml-pipelines-persistenceagent-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    containerfile: backend/src/Dockerfile.persistenceagent
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-persistenceagent/Dockerfile

  # ODH ML Pipelines API Server
  - name: odh-ml-pipelines-api-server-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    containerfile: backend/src/Dockerfile
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-api-server/Dockerfile

  # ML Pipelines Cacheserver
  - name: odh-ml-pipelines-cache-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-cache/Dockerfile

  # ML Pipelines Scheduled Workflow Server
  - name: odh-ml-pipelines-scheduledworkflow-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-scheduledworkflow/Dockerfile

  # ML Pipelines Artifact Manager
  - name: odh-ml-pipelines-artifact-manager-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines
    branch: master
    patch: rhods-operator-live/dockerfiles/odh-ml-pipelines-artifact-manager/Dockerfile

  # Data Science Pipelines Operator
  - name: odh-data-science-pipelines-operator-controller-container-live
    url: https://github.com/red-hat-data-services/data-science-pipelines-operator
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-data-science-pipelines-operator-controller/Dockerfile

  # ODH Modelmesh Serving Controller
  - name: odh-modelmesh-serving-controller-container-live
    url: https://github.com/red-hat-data-services/modelmesh-serving
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh-serving-controller/Dockerfile

  # ODH Model Controller
  - name: odh-model-controller-container-live
    url: https://github.com/red-hat-data-services/odh-model-controller
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-model-controller/Dockerfile

  # ODH Modelmesh Rest Proxy
  - name: odh-mm-rest-proxy-container-live
    url: https://github.com/red-hat-data-services/rest-proxy
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-mm-rest-proxy/Dockerfile

  # ODH Modelmesh Serving Runtime sidecar
  - name: odh-modelmesh-container-live
    url: https://github.com/red-hat-data-services/modelmesh.git
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh/Dockerfile

  # ModelMesh Runtime Adapter sidecar
  - name: odh-modelmesh-runtime-adapter-container-live
    url: https://github.com/red-hat-data-services/modelmesh-runtime-adapter
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-modelmesh-runtime-adapter/Dockerfile
    
  # TrustyAI Service
  - name: odh-trustyai-service-container-live
    url: https://github.com/red-hat-data-services/trustyai-explainability
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-trustyai-service/Dockerfile

  # TrustyAI Service
  - name: odh-trustyai-service-operator-container-live
    url: https://github.com/red-hat-data-services/trustyai-service-operator
    branch: main
    patch: rhods-operator-live/dockerfiles/odh-trustyai-service-operator/Dockerfile

```
</details>
